component/core/src/main/java/org/exoplatform/social/core/service/LinkProvider.java

```diff
@@ -59,6 +59,7 @@ public class LinkProvider {
 
   public static final String PROFILE_DEFAULT_AVATAR_URL = "/eXoSkin/skin/images/avatar/DefaultUserAvatar.png";
   public static final String SPACE_DEFAULT_AVATAR_URL = "/eXoSkin/skin/images/system/SpaceAvtDefault.png";
+  public static final String ROOM_DEFAULT_AVATAR_URL = "/chat/img/room-default.jpg";
```

Comment:
we can't add dependency in social about chat addon (dependency between bottom layer to top layer)

Reply:
> we can't add dependency in social about chat addon (dependency between bottom layer to top layer)

Ok, i will close this PR !